providers/efa/efadv.h

```diff
@@ -47,6 +47,20 @@ struct efadv_device_attr {
 int efadv_query_device(struct ibv_context *ibvctx,
 		       struct efadv_device_attr *attr);
 
+enum {
+	EFADV_AH_ATTR_AHN = 1 << 0,
+};
+
+struct efadv_ah_attr {
+	uint8_t version;
+	uint8_t reserved[7];
```

Comment:
Why do we need "reserved" fields in the middle of the newly introduced structure?

The same comment goes for the previous patch.

Reply:
it serves as padding as this struct is ABI

Reply:
Thanks, I forgot the padding thing.